code/modules/surgery/organs/external/_external_organ.dm

```diff
@@ -53,6 +53,15 @@
 	if(restyle_flags)
 		RegisterSignal(src, COMSIG_ATOM_RESTYLE, PROC_REF(on_attempt_feature_restyle))
 
+/obj/item/organ/external/Insert(mob/living/carbon/receiver, special = FALSE, movement_flags)
+	. = ..()
+	receiver.update_body_parts()
+
+/obj/item/organ/external/Remove(mob/living/carbon/organ_owner, special = FALSE, movement_flags)
```

Comment:
ditto (please use mob_remove() ), also please don't explicitly define FALSE in the params, so to be in line with the parent type